### PR TITLE
Fix force climbing DoAfter

### PR DESCRIPTION
**Pull request by Mr0maks**
## About the PR

DOAfter fix broken forced climbing. So there a fix for fix.

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

### CODE REVIEW

File: Content.Server/Climbing/ClimbSystem.cs [hunk 1]
```diff
@@ -124,7 +124,7 @@ private void OnClimbableDragDrop(EntityUid uid, ClimbableComponent component, re
             BreakOnStun = true,
             RaiseOnUser = user == entityToMove,
             RaiseOnTarget = user != entityToMove,
-            RaiseOnUsed = false,
```
Comment (ElectroJr) [inline Content.Server/Climbing/ClimbSystem.cs:127]: The event should only ever be raised directed at a single entity, not multiple. So the other arguments need to be changed to false.

**Review by ElectroJr — DISMISSED** [review]
.

File: Content.Server/Climbing/ClimbSystem.cs [hunk 2]
```diff
@@ -122,9 +122,9 @@ private void OnClimbableDragDrop(EntityUid uid, ClimbableComponent component, re
             BreakOnUserMove = true,
             BreakOnDamage = true,
             BreakOnStun = true,
-            RaiseOnUser = user == entityToMove,
-            RaiseOnTarget = user != entityToMove,
-            RaiseOnUsed = false,
+            RaiseOnUser = false,
+            RaiseOnTarget = false,
+            RaiseOnUsed = true
```
Comment (keronshb) [inline Content.Server/Climbing/ClimbSystem.cs:127]:
```suggestion
```

You can remove this since it's redundant.

**Review by keronshb — DISMISSED** [review]

**Comment by rbertoche** [comment]
nice

**Comment by rbertoche** [comment]
that change was really missing

**Pull request merged**